ci/prow/e2e_prebuilt_kernel_module_two_nodes

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+echo "Label the node so that it gets the module..."
+kubectl label node minikube wants-module=kmm_ci_a
+
+echo "Build and deploy the controller image..."
+./ci/github/build_controller
+
+echo "Build and deploy the DriverContainer..." 
+./ci/github/build_driver_container
+
+echo "Deploy KMMO..."
+make deploy KUSTOMIZE_CONFIG_DEFAULT=ci/install-ci
+
+echo "Wait until the KMMO Deployment is Available..."
+kubectl wait --for condition=Available deployments.apps -n kmm-operator-system kmm-operator-controller-manager --request-timeout=1m
+
+echo "Describe the Deployment and get its YAML if that failed..."
+kubectl describe deployments.apps -n kmm-operator-system kmm-operator-controller-manager
+kubectl get -o yaml deployments.apps -n kmm-operator-system kmm-operator-controller-manager
+
+echo "Check that the kmm_ci_a module is not loaded on the node..."
+          if minikube ssh -- lsmod | grep kmm_ci_a; then
+            echo "Unexpected lsmod output - the module should not be loaded"
+            exit 1
+          fi
+echo "Create one module for each node..."
+          # Node minikube gets module a
+          sed -e s/NAME_CHANGEME/kmm-ci-a/g \
+            -e s/KMOD_CHANGEME/kmm_ci_a/g \
+            -e "s/KVER_CHANGEME/$(uname -r)/g" \
+            ci/module-kmm-ci.template.yaml | tee module-kmm-ci-a.yaml
+
+          # Node minikube-m02 gets module b
+          sed -e s/NAME_CHANGEME/kmm-ci-b/g \
+            -e s/KMOD_CHANGEME/kmm_ci_b/g \
+            -e "s/KVER_CHANGEME/$(uname -r)/g" \
+            ci/module-kmm-ci.template.yaml | tee module-kmm-ci-b.yaml
+
+          kubectl apply -f module-kmm-ci-a.yaml -f module-kmm-ci-b.yaml
+
+
+echo "Label the first node to have it receive module a..."
+kubectl label node minikube wants-module=kmm_ci_a
+
+echo "Check that module a gets loaded on the first node..."
+          kubectl describe po
+          until minikube ssh -- lsmod | grep kmm_ci_a; do
+            sleep 3
+          done
+
+echo "Label the second node to have it receive module b..."
+kubectl label node minikube-m02 wants-module=kmm_ci_b
+
+echo "Check that module b gets loaded on the second node..."
+          until minikube ssh -n minikube-m02 -- lsmod | grep kmm_ci_b; do
+            sleep 3
+          done
+
+echo "Remove the wants-module label on the second node..."
+kubectl label node minikube-m02 wants-module-
+
+echo "Check that module b gets unloaded from the second node..."
+          until ! minikube ssh -n minikube-m02 -- lsmod | grep kmm_ci_b; do
+            sleep 3
+          done 
+
+echo "Verify that the DaemonSet gets garbage collected..."
+          # Cannot use kubectl wait because it will fail if there is no initial match
+          until [ $(kubectl get daemonsets.apps -l 'kmm.node.kubernetes.io/module.name=kmm-ci-b, kmm.node.kubernetes.io/role!=device-plugin' -o go-template='{{ len .items }}') -eq 0 ]; do
+            sleep 3
+          done
```

Comment:
Not sure we need that, if the kernel module was removed then we mush have cleaned the daemonset isn't it? This is implementation details IMO.

Reply:
How do we verify that garbage collection works? This is an important feature.

Reply:
Well, this is exactly what this `until` does: wait until the DaemonSets get garbage-collected. And it's from the existing workflow, so it's part of the test.

Reply:
Ohh sorry, I completely misunderstood that part.
We definitely needs it.

Sorry for the noise.

Reply:
How about moving this part to the "classic flow"? `e2e_prebuilt_kernel_module`